src/test/java/org/commcare/formplayer/tests/BaseTestClass.java

```diff
@@ -286,127 +268,18 @@ public void setUp() throws Exception {
-    }
-
-    private void setupRestoreFactoryMock() {
-        sessionSelectionsCache.clear();
-        RestoreFactoryAnswer answer = new RestoreFactoryAnswer(this.getMockRestoreFileName());
-        doAnswer(answer).when(restoreFactoryMock).getRestoreXml(anyBoolean());
-        Mockito.doReturn(false)
-                .when(restoreFactoryMock).isRestoreXmlExpired();
-        doAnswer(invocation -> {
-            String[] selections = (String[])invocation.getArguments()[0];
-            sessionSelectionsCache.add(String.join("|", selections));
-            return null;
-        }).when(restoreFactoryMock).cacheSessionSelections(any(String[].class));
-
-        doAnswer(invocation -> {
-            String[] selections = (String[])invocation.getArguments()[0];
-            return sessionSelectionsCache.contains(String.join("|", selections));
-        }).when(restoreFactoryMock).isConfirmedSelection(any(String[].class));
-    }
-
-    /*
-     * Setup mocking for the FormSessionService that allows saving and retrieving sessions.
-     * The 'persisted' sessions are cleared at the start of each test.
-     */
-    private void mockFormSessionService() {
-        sessionMap.clear();
-        doAnswer(new Answer<SerializableFormSession>() {
-            @Override
-            public SerializableFormSession answer(InvocationOnMock invocation) throws Throwable {
-                SerializableFormSession session =
-                        (SerializableFormSession)invocation.getArguments()[0];
-                if (session.getId() == null) {
-                    // this is normally taken care of by Hibernate
-                    ReflectionTestUtils.setField(session, "id", UUID.randomUUID().toString());
-                }
-                sessionMap.put(session.getId(), session);
-                return session;
-            }
-        }).when(formSessionService).saveSession(any(SerializableFormSession.class));
-
-        when(formSessionService.getSessionById(anyString())).thenAnswer(
-                new Answer<SerializableFormSession>() {
-                    @Override
-                    public SerializableFormSession answer(InvocationOnMock invocation)
-                            throws Throwable {
-                        String key = (String)invocation.getArguments()[0];
-                        if (sessionMap.containsKey(key)) {
-                            return sessionMap.get(key);
-                        }
-                        throw new FormNotFoundException(key);
-                    }
-                });
 
-        doAnswer(new Answer<Void>() {
-            @Override
-            public Void answer(InvocationOnMock invocation) throws Throwable {
-                String key = (String)invocation.getArguments()[0];
-                sessionMap.remove(key);
-                return null;
-            }
-        }).when(formSessionService).deleteSessionById(anyString());
+        new SQLiteProperties().setDataDir("testdbs/");
```

Comment:
Do we need it here as well when we are doing the same in `TestContext` ?

Reply:
no I don't think so

Reply:
turns out it is needed - I'm not sure why